wgpu/tests/common/mod.rs

```diff
@@ -358,7 +358,7 @@ fn initialize_adapter() -> (Adapter, SurfaceGuard) {
         let canvas = create_html_canvas();
 
         let surface = instance
-            .create_surface_from_canvas(&canvas)
```

Comment:
This is a breaking change, so should be highlighted in the major changes section of the changelog

Reply:
Done, I (hopefully) improved the changelog a bit.